backend/src/main/resources/db/migration/V1.28.0__NOTIF-190_cronjob_timestamp.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE cronjob_run (
+    last_run TIMESTAMP,
```

Comment:
Sorry, I forgot this earlier 😄
```suggestion
    last_run TIMESTAMP NOT NULL,
```